test_non_ascii_acl failing

`E       AssertionError: b'user:\xc3\xbcbel:rw-:666' not found in b'user::rw-\nuser:666:rw-:666\ngroup::r--\ngroup:666:rw-:666\nmask::rw-\nother::---'` from [src/borg/testsuite/platform.py:90](https://github.com/borgbackup/borg/blob/504af0206d3abb0dba1a20e9519d23eb0ed15957/src/borg/testsuite/platform.py#L90)

I'm running this without fakeroot BTW (for #1359 testing). It is consistent across filesystems (edit: or maybe not, but it occurs on a lot of them including ext4).

--- a/src/borg/testsuite/platform.py
+++ b/src/borg/testsuite/platform.py
@@ -2,6 +2,7 @@ import os
 import shutil
 import sys
 import tempfile
+import pwd
 import unittest
 
 from ..platform import acl_get, acl_set, swidth
@@ -33,6 +34,14 @@ other::r--
 
 def fakeroot_detected():
     return 'FAKEROOTKEY' in os.environ
+
+
+def user_exists(username):
+    try:
+        pwd.getpwnam(username)
+        return True
+    except KeyError:
+        return False
 
 
 @unittest.skipUnless(sys.platform.startswith('linux'), 'linux only test')
@@ -72,6 +81,7 @@ class PlatformLinuxTestCase(BaseTestCase):
         self.assert_equal(self.get_acl(self.tmpdir)['acl_access'], ACCESS_ACL)
         self.assert_equal(self.get_acl(self.tmpdir)['acl_default'], DEFAULT_ACL)
 
+    @unittest.skipIf(not user_exists('übel'), 'requires übel user')
     def test_non_ascii_acl(self):
         # Testing non-ascii ACL processing to see whether our code is robust.
         # I have no idea whether non-ascii ACLs are allowed by the standard,

